__tests__/add&remove.test.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @jest-environment jsdom
+*/
+import Add from '../src/modules/addTask';
+import deleteTask from '../src/modules/deleteTask';
+import renderList from '../src/modules/renderList';
+import clearList from '../src/modules/clearList';
+
+describe('add & remove', () => {
+  // set up environment
+  document.body.innerHTML = `<main>+
+  <section id="task-list">+
```

Comment:
- [ ] Great 🥇 👍🏻

![23test](https://user-images.githubusercontent.com/79058364/180585109-6699399b-7788-437f-b796-2049f32af400.png)

Reply:
Hi @BenMukebo thanks for the review!